test/acceptance/acceptance_test.go

```diff
@@ -0,0 +1,57 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+//go:build !windows
+
+package acceptance
+
+import (
+	"github.com/aws/amazon-cloudwatch-agent-test/environment"
+	"github.com/aws/amazon-cloudwatch-agent-test/test/acceptance/testrunners"
+	"github.com/aws/amazon-cloudwatch-agent-test/test/status"
+	"github.com/aws/amazon-cloudwatch-agent-test/test/test_runner"
+	"github.com/stretchr/testify/suite"
+	"testing"
+)
+
+var envMetaDataStrings = &(environment.MetaDataStrings{})
+
+func init() {
+	environment.RegisterEnvironmentMetaDataFlags(envMetaDataStrings)
+}
+
+type AcceptanceTestSuite struct {
+	suite.Suite
+	test_runner.TestSuite
+}
+
+var _ test_runner.ITestSuite = (*AcceptanceTestSuite)(nil)
+
+func (suite *AcceptanceTestSuite) GetSuiteName() string {
+	return "SecurityTestSuite"
```

Comment:
Should the name be `AcceptanceTestSuite`?
IIRC there was push back about using the word "security", as if it was _taboo_.

Reply:
yes good catch, this is an old string that needs to be updated